code/modules/mob/mob_defines.dm

```diff
@@ -151,6 +151,7 @@
 
 	var/job = null//Living
 	var/megavend = 0		//determines if this ID has claimed their megavend stache
+	var/newarrival = 0 //For shuttle management
```

Comment:
Only carbon/humans can be spawned. Don't use a general mob variable. Technically megavend should also be moved at this rate.